src/test/resources/com/puppycrawl/tools/checkstyle/checks/modifier/redundantmodifier/InputRedundantModifierWithStrictfpAsRedundant.java

```diff
@@ -0,0 +1,13 @@
+/*
+RedundantModifier
+tokens = STRICTFP, METHOD_DEF
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.modifier.redundantmodifier;
+
+public class InputRedundantModifierWithStrictfpAsRedundant {
+    strictfp void method(){} // violation
```

Comment:
It would be good to extend test cases a bit:

```
 ➜  src cat Test.java
public strictfp class Test {} // violation

strictfp interface MyInterface {} // violation

strictfp enum MyEnum {} // violation

strictfp record MyRecord() {} // violation

abstract strictfp class MyClass {} // violation

abstract strictfp interface MyStrictFPInterface { // 2x violation
    public static strictfp enum MyInnerEnum {} // 3x violation
}

final class OtherClass {
    final strictfp void m1() {} // 2x violation
}
➜  src javac Test.java
➜  src 
```

What do you think, @AkMo3 ? https://www.geeksforgeeks.org/strictfp-keyword-java/

Reply:
Done.

Reply:
Resolved, issue opened at https://github.com/checkstyle/checkstyle/issues/10806 for missing violation for inner enum.